distributed/client.py

```diff
@@ -785,7 +786,21 @@ def __repr__(self):
         else:
             return "<%s: not connected>" % (self.__class__.__name__,)
 
-    def _repr_html_(self):
+    @property
+    def _dashboard_address(self):
+        scheduler, info = self._maybe_get_scheduler_and_info()
+        if info and "bokeh" in info["services"]:
```

Comment:
Note that this is changing to `"dashboard"` in https://github.com/dask/distributed/pull/2724